src/logging.cc

```diff
@@ -476,14 +497,24 @@ class LogFileObject : public base::Logger {
   uint32 bytes_since_flush_{0};
   uint32 dropped_mem_length_{0};
   uint32 file_length_{0};
-  unsigned int rollover_attempt_;
+  uint32 rollover_attempt_;
   int64 next_flush_time_{0};  // cycle count at which to flush log
   WallTime start_time_;
+  std::multiset<Filetime> file_list_;
```

Comment:
I'm not sure why a `std::multiset` is needed here. Can a file be associated with a multiple timestamps? I don't see how this is possible. Either way, a `multiset` would rather memory inefficient here, since the same file name would be potentially stored multiple times.

Please drop the `Filetime` structure and use an associative container. I guess a `std::map` is sufficient here.